storage/src/main/java/zipkin2/storage/kafka/KafkaSpanStore.java

```diff
@@ -111,10 +112,12 @@
 
     @Override public List<String> query() {
       List<String> serviceNames = new ArrayList<>();
-      serviceStore.all().forEachRemaining(keyValue -> {
-        // double check service names are unique
-        if (!serviceNames.contains(keyValue.value)) serviceNames.add(keyValue.value);
-      });
+      try (KeyValueIterator<String, String> all = serviceStore.all()) {
```

Comment:
sheesh do we allow dupes? sounds like we should tighten up that javadoc

Reply:
yeap, that surprise me on one of my deployments where I started to get duplicate service names. Will create an issue to double check this.

Reply:
#35